frontend/src/metabase-types/api/data-app.ts

```diff
@@ -2,11 +2,19 @@ import { Collection, RegularCollectionId } from "./collection";
 import { ClickBehavior } from "./click-behavior";
 import {
   BaseDashboardOrderedCard,
+  Dashboard,
   DashboardParameterMapping,
 } from "./dashboard";
 import { WritebackAction } from "./writeback";
 
 export type DataAppId = number;
+export type DataAppPageId = Dashboard["id"];
+
+export interface DataAppNavItem {
+  page_id: DataAppPageId;
+  indent?: number;
```

Comment:
are we just going to define indents instead of parents? this feels like it would prevent us from expand/collapse behavior or reordering groups of things? Maybe i'm reading too much into this.

Reply:
Good call. This structure is pretty easier to reason about compared to other options we considered. Let's see how far we can get with it. We can still compute parents and groups of pages out of `indent`, so let's see if it works well